src/com/lftechnology/java/training/srijan/basics/genericclass/GenericService.java

```diff
@@ -0,0 +1,9 @@
+package com.lftechnology.java.training.srijan.basics.genericclass;
+
+import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)